src/lib.rs

```diff
@@ -61,18 +67,26 @@ pub struct KeyPair {
 }
 
 #[derive(Debug)]
-/// Message type for which cryptographic traits are defined.
+/// Message struct for which cryptographic traits are defined.
 pub struct Message {
+    // Input message
```

Comment:
Any publicly available fields like this need a doc level comment with 3 slashes like this:

```rust
/// Input message
```